java/statestore/src/test/java/sleeper/statestore/s3/S3FileReferenceStoreIT.java

```diff
@@ -234,6 +228,52 @@ void shouldSplitFilesInDifferentPartitions() throws Exception {
                             splitFile(file2, "RR"));
         }
 
+        @Test
+        void shouldOnlyPerformOneLevelOfSplits() throws Exception {
```

Comment:
Is there a reason these tests have not been added to the DynamoDBFileReferenceStoreIT?

Reply:
This PR was originally combined with https://github.com/gchq/sleeper/pull/1705, I added the tests to the DynamoDBFileReferenceStoreIT class in that PR, I'll see about getting that commit in this PR

Reply:
That might not be necessary - I'd just noticed they were added to https://github.com/gchq/sleeper/pull/1705/, we can leave them to be added there if it's easier.

Reply:
Its ok, I've included that commit in this PR now